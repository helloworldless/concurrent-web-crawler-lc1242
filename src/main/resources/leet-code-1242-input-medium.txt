["http://cno.bytij.top/hgzg","http://cno.bytij.top/bchw","http://cno.bytij.top/pshw","http://cno.bytij.top/nufc","http://cno.bytij.top/vqfc","http://cno.bytij.top/ngls","http://cno.bytij.top/hwpg","http://cno.bytij.top/xujw","http://cno.bytij.top/rylq","http://cno.bytij.top/zejm","http://cno.bytij.top/dubk","http://cno.bytij.top/xsbc","http://cno.bytij.top/tsfw","http://cno.bytij.top/delk","http://cno.bytij.top/dqzs","http://cno.bytij.top/hupm","http://cno.bytij.top/rufy","http://cno.bytij.top/xkls","http://cno.bytij.top/jure","http://cno.bytij.top/vipy","http://cno.bytij.top/fobu","http://cno.bytij.top/dkbg","http://cno.bytij.top/pqta","http://cno.bytij.top/dspg","http://cno.bytij.top/vina","http://cno.bytij.top/fefk","http://cno.bytij.top/tcti","http://cno.bytij.top/nyvg","http://bkj.grihw.xyz/fkpu","http://ubu.nqrgn.xyz/sryb","http://lmr.wberi.top/feba","http://ony.bqpkd.xyz/ejgz","http://fof.stsxi.info/tkbk","http://cre.pqhkl.org/mrkt","http://lqv.etgnm.tech/bcji","http://wra.lgrip.ink/ijov","http://jox.mvkpu.xyz/tarm","http://src.jqbmb.xyz/ebyv","http://jin.odehs.red/vejq","http://wrc.pqpyp.ink/gfet"]
[[0,32],[0,23],[0,18],[0,25],[0,20],[0,19],[0,6],[0,29],[0,16],[0,39],[0,18],[0,17],[0,28],[0,11],[0,14],[0,5],[0,16],[0,39],[1,25],[1,36],[1,11],[1,6],[1,37],[1,16],[1,31],[1,10],[1,33],[2,35],[2,14],[2,13],[2,0],[2,39],[2,18],[2,33],[2,28],[2,11],[2,30],[3,24],[3,7],[3,2],[3,9],[3,12],[3,3],[3,6],[3,21],[3,16],[3,31],[3,26],[3,25],[3,4],[3,35],[4,37],[4,0],[4,39],[4,26],[4,1],[4,28],[4,19],[4,30],[4,13],[4,16],[4,39],[5,25],[5,28],[5,3],[5,30],[5,21],[7,34],[7,17],[7,4],[7,27],[7,30],[7,13],[7,0],[7,15],[7,18],[7,1],[7,4],[7,11],[7,38],[7,37],[7,0],[7,39],[7,18],[7,9],[7,4],[8,38],[8,37],[8,24],[8,15],[8,18],[10,19],[10,14],[11,0],[11,15],[12,33],[13,11],[13,22],[13,29],[13,16],[13,31],[13,18],[13,1],[13,20],[13,19],[13,14],[14,0],[14,7],[14,2],[14,1],[14,36],[14,19],[14,14],[14,37],[14,0],[14,39],[15,9],[15,36],[15,27],[15,30],[15,21],[15,32],[15,31],[15,34],[15,17],[15,12],[15,19],[15,30],[15,37],[15,32],[15,23],[15,26],[15,1],[16,19],[16,22],[16,5],[16,24],[16,7],[16,10],[16,1],[16,28],[16,19],[16,14],[16,29],[16,24],[16,15],[16,18],[16,9],[16,20],[16,35],[16,6],[17,16],[17,7],[18,25],[18,20],[18,11],[18,22],[18,21],[18,24],[18,31],[18,26],[18,33],[18,28],[18,35],[18,22],[18,29],[18,32],[18,23],[18,10],[18,33],[19,3],[19,6],[20,8],[20,39],[20,26],[20,9],[20,28],[20,11],[21,21],[21,16],[21,39],[21,2],[21,33],[21,28],[21,27],[21,22],[21,21],[21,32],[21,39],[22,1],[22,12],[22,19],[22,6],[22,21],[22,16],[22,7],[22,10],[22,17],[22,36],[22,3],[22,30],[22,13],[23,31],[23,34],[23,33],[23,20],[23,11],[23,14],[23,21],[23,8],[24,18],[24,33],[24,4],[25,38],[25,29],[25,32],[25,39],[25,34],[25,33],[25,12],[25,27],[25,22],[25,13],[25,24],[25,15],[25,10],[26,12],[26,35],[26,38],[26,5],[26,0],[26,39],[26,34],[26,1],[26,12],[26,11],[26,22],[26,21],[27,31],[27,18],[27,9],[27,4],[27,3],[27,30],[27,29],[27,24],[28,10],[28,17],[28,20],[29,30],[30,16],[30,7],[31,33],[31,36],[31,35],[31,38],[31,5],[31,0],[31,15],[31,26],[31,17],[31,28],[31,27],[31,6],[31,21],[31,8],[31,15],[31,2],[31,9],[32,35],[32,30],[32,37],[32,0],[32,15],[32,2],[32,9],[32,28],[32,11],[32,30],[33,16],[33,31],[33,2],[33,9],[33,12],[33,27],[33,38],[33,21],[33,16],[33,39],[33,10],[33,25],[33,12],[33,35],[33,14],[33,29],[33,24],[33,23],[34,25],[34,28],[34,35],[34,30],[34,29],[35,23],[35,26],[35,33],[35,28],[35,19],[35,6],[35,21],[35,0],[35,7],[35,18],[35,17],[35,20],[35,11],[35,38],[35,29],[35,8],[36,18],[36,25],[36,20],[36,27],[36,6],[36,5],[36,16],[36,15],[36,18],[36,17],[36,28],[37,22],[37,29],[37,16],[37,39],[37,18],[38,20],[38,35],[38,22],[38,37],[40,15],[40,2],[40,1],[40,20],[40,11],[40,14],[40,21],[40,24],[40,39],[40,10],[40,1],[40,12],[40,3],[40,38],[40,21],[40,8],[40,7],[40,18],[40,1],[40,28],[40,3],[40,38],[40,21],[40,0],[40,7],[40,26],[40,9],[40,28],[40,27],[40,6],[40,37],[40,8]]
"http://cno.bytij.top/nyvg"