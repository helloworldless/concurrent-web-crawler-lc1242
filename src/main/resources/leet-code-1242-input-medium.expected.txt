["http://cno.bytij.top/tcti","http://cno.bytij.top/vqfc","http://cno.bytij.top/dqzs","http://cno.bytij.top/pqta","http://cno.bytij.top/rylq","http://cno.bytij.top/dubk","http://cno.bytij.top/zejm","http://cno.bytij.top/vipy","http://cno.bytij.top/hupm","http://cno.bytij.top/vina","http://cno.bytij.top/nufc","http://cno.bytij.top/xujw","http://cno.bytij.top/pshw","http://cno.bytij.top/fobu","http://cno.bytij.top/hgzg","http://cno.bytij.top/hwpg","http://cno.bytij.top/dkbg","http://cno.bytij.top/bchw","http://cno.bytij.top/fefk","http://cno.bytij.top/xsbc","http://cno.bytij.top/dspg","http://cno.bytij.top/tsfw","http://cno.bytij.top/rufy","http://cno.bytij.top/ngls","http://cno.bytij.top/xkls","http://cno.bytij.top/delk","http://cno.bytij.top/jure","http://cno.bytij.top/nyvg"]